server/api/lib/mavat/planInstructions/parser/chartThreeOne.js

```diff
@@ -0,0 +1,95 @@
+const { chartToArrayBuilder } = require('./chartToArrayBuilder');
+const { getFromArr } = require('./parsersUtils');
+const log = require('../../../log');
+
+// this function look for the correct columns for the given headers, and returns a factory embedded with these findings
+const rowAbstractFactoryCharts3_1_without_change = (firstPageOfTable, headersStartIndex) => {
+	// clean the headers
+	firstPageOfTable = firstPageOfTable.map(row => row.map(cell => cell.replace(/\n/g, ' ')
+		.replace(/ {2}/g, ' ')
+		.trim()));
+
+	if (headersStartIndex === -1) {
+		// it's an error
+		log.error('didn\'t find headers for some chart 3.1_without_change');
```

Comment:
plan id

Reply:
Same thing as the comment on the log.error on 1.6